templates/zerver/dev_tools.html

```diff
@@ -0,0 +1,43 @@
+{% extends "zerver/portico.html" %}
+
+{# Login page. #}
+{% block portico_content %}
+<div id="devtools-page">
+    <h2>Useful development URLs</h2>
+    <p>
+        {% trans %}
+        Below is a list of useful URLs that provide information useful in development.
```

Comment:
Maybe, "Below is a list of useful tools and data sets available only in the Zulip development environment that are often useful when contributing to Zulip.  "